tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -454,7 +454,7 @@
       "MinVersion": "2.0.0",
       "SkipAlpine": true
     }, {
-      "MaxVersionExclusive": "2.37.0",
+      "MaxVersionExclusive": "2.38.1",
```

Comment:
Hmm? I'm not sure what you mean, we're able to run versions like 2.48.0, 2.52.0, etc. on ARM64 so I think this exclusion filter is set appropriately

Reply:
Oh I think see what you're saying now, that the SpecificVersions only does 2.29.0, 2.30.0, and then the latest 2.x. Umm, I kinda like having the 2.38.1 version being declared here in case we ever decide to specifically test a 2.x version in the future

Reply:
oh, no, I assumed all the later versions didn't work either, never mind, disregard 😄